test/lib-http-proxy-passes-web-incoming-test.js

```diff
@@ -287,6 +287,41 @@ describe('#createProxyServer.web() using own http server', function () {
     }, function() {}).end();
   });
 
+  it('should proxy the request with custom timeout errors (proxyTimeoutCustomError)', function(done) {
+    var proxy = httpProxy.createProxyServer({
+      target: 'http://127.0.0.1:45002',
+      proxyTimeout: 100,
+      proxyTimeoutCustomError: true
+    });
+
+    require('net').createServer().listen(45002);
+
+    var proxyServer = http.createServer(requestHandler);
+
+    var started = new Date().getTime();
+    function requestHandler(req, res) {
+      proxy.once('error', function (err, errReq, errRes) {
+        proxyServer.close();
+        expect(err).to.be.an(Error);
+        expect(errReq).to.be.equal(req);
+        expect(errRes).to.be.equal(res);
+        expect(new Date().getTime() - started).to.be.greaterThan(99);
```

Comment:
what about adding the `expect(new Date().getTime() - started).to.be.above(99);
`. Will it make a difference to this line or to the whole code